media/livestream/livestream_test.go

```diff
@@ -517,6 +525,64 @@ func TestChannelEvents(t *testing.T) {
 		}
 	})
 
+	// Tests for clips
+	// Send a test stream for the clip.
+	cmd := exec.Command("ffmpeg", "-re", "-f", "lavfi", "-t", "45", "-i",
```

Comment:
issue: I have concerns about running arbitrary processes from within tests. @muncus PTAL.

1. Let's isolate this test into its own file. I want to keep this test hermeneutically sealed from other tests as best as possible.
2. What exactly is the point of executing this command? The bytes returned from the output are thrown away.

Reply:
IIUC, this command is generating a video stream. Can we do this with a static file instead?
I'd prefer to avoid adding `ffmpeg` to our testing images, as it has a history of vulnerabilities and could increase ongoing maintenance requirements.